staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -1514,3 +1519,56 @@ message ReopenContainerLogRequest {
 
 message ReopenContainerLogResponse{
 }
+
+// Common options used for checkpointing and restoring.
+message CheckpointRestoreOptions {
+    // Location of the checkpoint archive used for export/import
+    string archive_location = 1;
+    // Instead of using a local checkpoint archive use a OCI image
+    // to store the checkpoint.
+    bool archive_is_oci_image = 2;
+}
+
+message CheckpointContainerOptions {
+    CheckpointRestoreOptions common_options = 1;
+    // Leave the container running after checkpointing
+    bool leave_running = 2;
```

Comment:
This could be the default for the forensic use case so can be dropped.

Reply:
dropped